mysql-test/suite/rocksdb.rpl/t/multiclient_2pc.test

```diff
@@ -23,7 +19,7 @@ connect (con2, localhost, root,,);
 # has been recovered and commited
 connection con1;
 --echo 'con1'
---exec echo "restart" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--exec echo "$restart_parameters" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
 SET SESSION debug="d,crash_commit_after_log";
 SET DEBUG_SYNC='rocksdb.prepared SIGNAL parked WAIT_FOR go';
 --error 0,2013
```

Comment:
I tried CR_SERVER_LOST and it doesn't work since it is a client error code. It seems 5.7 does not yet support symbolic name matching for client errors, only numerical. According to the docs [here](https://dev.mysql.com/doc/mysqltest/2.0/en/writing-tests-expecting-errors.html), 8.0 will support symbolic names for client error codes. So I'm just removing the 0